Makefile

```diff
@@ -58,32 +59,32 @@ TTY := -t
 KUSTOMIZE := $(abspath $(TOOLS_BIN_DIR)/kustomize)
 kustomize: $(KUSTOMIZE) ## Build a local copy of kustomize
 $(KUSTOMIZE): # Build kustomize from tools folder.
-	cd $(TOOLS_DIR) && go build -mod=vendor -tags=tools -o $(KUSTOMIZE) sigs.k8s.io/kustomize/kustomize/v4
+	go install sigs.k8s.io/kustomize/kustomize/v5@v5.1.0
 
 ENVSUBST := $(abspath $(TOOLS_BIN_DIR)/envsubst)
 envsubst: $(ENVSUBST) ## Build a local copy of envsubst
 $(ENVSUBST): $(TOOLS_DIR)/go.mod # Build envsubst from tools folder.
-	cd $(TOOLS_DIR) && go build -mod=vendor -tags=tools -o $(ENVSUBST) github.com/drone/envsubst/v2/cmd/envsubst
+	go install github.com/drone/envsubst/v2/cmd/envsubst@latest
```

Comment:
we should use a specific version

Reply:
There is no version mentioned here in the [go pkg](https://pkg.go.dev/github.com/drone/envsubst/v2/cmd/envsubst?tab=versions)

Reply:
but we could use [root pkg version](https://pkg.go.dev/github.com/drone/envsubst?tab=versions)

Reply:
yes sounds good!

Reply:
```shell
go install github.com/drone/envsubst/v2/cmd/envsubst@v1.0.3
go: github.com/drone/envsubst/v2/cmd/envsubst@v1.0.3: module github.com/drone/envsubst@v1.0.3 found, but does not contain package github.com/drone/envsubst/v2/cmd/envsubst
```

I think we need a version at cmd pkg, and that we don't have